schemas/common/selfservice.schema.json

```diff
@@ -0,0 +1,170 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
```

Comment:
Merge from master, we switched to CC BY 4.0 license

Reply:
Done

Reply:
Because the line on which the comment was made is the same in both licenses GitHub does not think there i sa change and hence as not auto resolved this change request.